src/js/rx/components/GlossaryModal.jsx

```diff
@@ -23,8 +24,12 @@ class GlossaryModal extends React.Component {
             <div className="va-modal-body rx-modal-body">
               <GlossaryList terms={this.props.content}/>
               <div className="va-modal-button-group cf">
-                <button type="button" onClick={this.handlerCloseModal}>Close</button>
-                <a href="/rx/glossary/">See all status definitions</a>
+                <button type="button" onClick={this.handleCloseModal}>Close</button>
+                <Link
+                    to="/rx/glossary"
+                    onClick={this.props.onCloseModal}>
```

Comment:
I didn't pass `handleCloseModal` here because the method does a `preventDefaults`, which stops the link from going to the glossary.